openc3/lib/openc3/migrations/20250108060000_news_feed.rb

```diff
@@ -0,0 +1,16 @@
+require 'openc3/utilities/migration'
+require 'openc3/models/scope_model'
+require 'openc3/models/setting_model'
+
+module OpenC3
+  class NewsFeed < Migration
+    def self.run
+      setting = SettingModel.get(name: 'news_feed')
+      SettingModel.set({ name: 'news_feed', data: true }, scope: 'DEFAULT') unless setting
```

Comment:
I think this migration has issues because I'm storing `true/false` in there. So this will basically always turn it on. Not sure how to check for existence vs true/false.

Reply:
We have other true/false settings right?

Reply:
Oh I see the always will become true thing.  You don't really need to unless part, as this should only run on migration and in that scenario the setting really shouldn't have ever existed before.